pandas/core/generic.py

```diff
@@ -5716,24 +5716,86 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Return copy of the input with values below given value(s) trimmed.
```

Comment:
Let's use "caller" instead of "input". FWIW I guess it's not technically true that it returns a copy of the caller because `inplace` would modify directly. Any idea on how to make this statement more accurate?

Reply:
Something along the line of

Return a copy of the caller (or the caller itself if inplace == True) with values below given `threshold` trimmed.

?

Reply:
Hmm well I don't think you need to get in the discussion of return values since that's documented later in the docstring. First line is supposed to be very simple so I would mirror what's there for `clip` and say something like `Trim the caller's values below a given threshold`

Reply:
Ok. Since Series.clip_lower uses the same docstring (the function is defined in generic.py) I thought that I should change the "See Also" description for Series.clip_lower to be the same. But when I tested the generation of the HTML I realized that if we use the "See Also" as it is right now, for Series.clip_lower we will not have a reference to DataFrame.clip_lower and it will have a reference to itself.

Does this make sense? Is there an elegant way to solve it?

Reply:
Yes that makes sense. Typically in this case you could use the `@Substitution` decorator to which you'll see some other functions using in that same module.

That said, since your method is currently just inherited by `Series` and `DataFrame` and not necessarily overriden, I don't think there's a good way without making "code updates" to implement Substitution on your docstring. Unless you can think of a better way looking at it, I'd suggest you remove the `clip_lower` reference in See Also and open up a separate issue to try and make substitution work here

Reply:
Nope, I can also only think of overriding in order for it to work. But is it worth the effort just to make the "See Also" complete? Would a reference to just clip_lower (without DataFrame. or Series. prefixes) be a good enough option?

I'll sleep through it to see if I can think of something better.

Reply:
I would say just remove but if you come up with a better idea then by all means